src/zenml/integrations/airflow/orchestrators/airflow_orchestrator.py

```diff
@@ -205,21 +205,28 @@ def up(self) -> None:
 
         from airflow.cli.commands.standalone_command import StandaloneCommand
 
-        command = StandaloneCommand()
-        # Run the daemon with a working directory inside the current
-        # zenml repo so the same repo will be used to run the DAGs
-        daemon.run_as_daemon(
-            command.run,
-            pid_file=self.pid_file,
-            log_file=self.log_file,
-            working_directory=zenml.io.utils.get_zenml_dir(),
-        )
-
-        while not self.is_running:
-            # Wait until the daemon started all the relevant airflow processes
-            time.sleep(0.1)
-
-        self._log_webserver_credentials()
+        try:
+            command = StandaloneCommand()
+            # Run the daemon with a working directory inside the current
+            # zenml repo so the same repo will be used to run the DAGs
+            daemon.run_as_daemon(
+                command.run,
+                pid_file=self.pid_file,
+                log_file=self.log_file,
+                working_directory=zenml.io.utils.get_zenml_dir(),
+            )
+            while not self.is_running:
+                # Wait until the daemon started all the relevant airflow processes
+                time.sleep(0.1)
+            self._log_webserver_credentials()
+        except Exception as e:
+            self.down()
+            logger.error(e)
+            logger.error(
+                "An error occurred while starting the Airflow daemon."
```

Comment:
the nittiest of nitpicks here, but I guess you could put a trailing space at the end of this string? since all these strings get concatenated